src/meta/src/hummock/compactor_manager.rs

```diff
@@ -228,19 +234,18 @@ impl CompactorManagerInner {
         ret
     }
 
-    pub fn get_expired_tasks(&self, interval_sec: Option<u64>) -> Vec<CompactTask> {
-        let interval = interval_sec.unwrap_or(0);
-        let now: u64 = SystemTime::now()
+    pub fn get_expired_tasks(&self) -> Vec<CompactTask> {
+        let heartbeat_expiry_ts: u64 = SystemTime::now()
             .duration_since(SystemTime::UNIX_EPOCH)
             .expect("Clock may have gone backwards")
             .as_secs()
-            - interval;
-        Self::get_heartbeat_expired_tasks(&self.task_heartbeats, now)
+            - self.heartbeat_expiry_seconds;
+        Self::get_heartbeat_expired_tasks(&self.task_heartbeats, heartbeat_expiry_ts)
     }
 
     fn get_heartbeat_expired_tasks(
         task_heartbeats: &HashMap<HummockCompactionTaskId, TaskHeartbeat>,
-        now: u64,
+        heartbeat_expiry_ts: u64,
```

Comment:
is it a typo ?

Reply:
It's not a typo. After we distinguish process and heartbeat timeout, the background thread only needs to check expired_task based on heartbeat_expiry_ts